city_scrapers/spiders/chi_northern_il_university.py

```diff
@@ -0,0 +1,129 @@
+            for one_day_links in title.xpath('.//following-sibling::ul[1]/li'):
+                href = one_day_links.xpath('.//@href').get()
+                name = one_day_links.xpath('.//text()').extract()
+                if href is None:
+                    link = ''
+                else:
+                    link = response.urljoin(href)
+                self.link_date_map.setdefault(new_date_obj, []).append({
+                    "title": ''.join(name),
+                    "href": link
+                })
+        yield response.follow(
+            "https://www.neiu.edu/about/board-of-trustees/calendar-of-meetings",
+            callback=self._parse_detail
+        )
+
+    def _parse_detail(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        items_list = response.xpath(
+            '//div[@class="field field--name-field-generic-body field--type-text-long \
+field--label-hidden field--item"]'
+        )
+        year_list = items_list.xpath('.//h2')
+        for years in year_list.extract():
+            year = re.findall(r'\d{4}', years)[0]
+            items = year_list.xpath('.//following-sibling::ul[1]/li').extract()
+            for item in items:
+                start = self._parse_start(item, year)
+                meeting = Meeting(
+                    title=self._parse_title(item),
+                    description='',
+                    classification=self._parse_classification(item),
+                    start=start,
+                    end=None,
+                    all_day=False,
+                    time_notes="See agenda for meeting time",
+                    location=self._parse_location(item),
+                    links=self.link_date_map[start],
+                    source=self._parse_source(response),
+                )
+                meeting["status"] = self._get_status(meeting)
+                meeting["id"] = self._get_id(meeting)
+                yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        title_obj = item.split('-')
+        title = str()
+        for i in range(1, len(title_obj)):
+            title = title + title_obj[i]
+        title = re.sub(r'<strong>|</strong>|<li>|</li>|\xa0', ' ', title)
+        return title.strip(" ")
+
+    def _parse_classification(self, title):
+        """Parse or generate classification from allowed options."""
+        if 'committee' in title.lower():
+            return COMMITTEE
+        if "meeting" in title.lower():
+            return BOARD
+        return NOT_CLASSIFIED
+
+    def _parse_start(self, item, year):
+        """Parse start datetime as a naive datetime object."""
+        date = re.sub(r'<li>|,|\xa0|\*|\*\*|<strong>|</strong>', '', item.split("-")[0]).strip(" ")
```

Comment:
Mentioned above, but this should use the pre-parsed string from the loop and also not replace the time since we don't know it